### ISSUE TITLE
Missing libgvplugin_pango

### ISSUE THREAD

**Issue by johanneskoester** [conda-forge/graphviz-feedstock#18]
I get the following error when using the dot command:

```
Warning: Could not load "/home/johannes/.local/opt/miniconda3/envs/snakemake-tutorial/lib/graphviz/libgvplugin_pango.so.6" - file not found
Warning: Could not load "/home/johannes/.local/opt/miniconda3/envs/snakemake-tutorial/lib/graphviz/libgvplugin_pango.so.6" - file not found
Format: "jpg" not recognized. Use one of: bmp canon cmap cmapx cmapx_np dot eps fig gv ico imap imap_np ismap jpe jpeg jpg pdf pic plain plain-ext png pov ps ps2 svg svgz tif tiff tk vml vmlz xdot xdot1.2 xdot1.4
```

This is on fedora linux 26.

**Comment by johanneskoester** [comment]
@ocefpaf, since you are one of the maintainers, any insight on this?

**Comment by ocefpaf** [comment]
No idea. I kind don't really use `graphviz` a lot. Ping @conda-forge/graphviz team.

**Comment by ocefpaf** [comment]
BTW, if you can create a test and send it as a PR we can check if a re-build fixes this.

**Comment by ccordoba12** [comment]
@johanneskoester, what were you doing when you got that error?

**Comment by rasmusagren** [comment]
I have the same error on Ubuntu. Here is a Dockerfile to reproduce it.

```
FROM ubuntu:latest

RUN apt-get update && apt-get -y dist-upgrade
RUN apt-get install -y --no-install-recommends bzip2 wget

RUN wget --no-check-certificate https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
    bash Miniconda3-latest-Linux-x86_64.sh -bf && \
    rm Miniconda3-latest-Linux-x86_64.sh
ENV PATH="/root/miniconda3/bin:${PATH}"

RUN conda install -c conda-forge graphviz=2.38.0

CMD dot
```

This gives:
```
$ docker run --rm docker_graphviz
Warning: Could not load "/root/miniconda3/lib/graphviz/libgvplugin_pango.so.6" - file not found
```

Seems like an X libraries thing..

```
root@e87e82132286:/home# ldd /root/miniconda3/lib/graphviz/libgvplugin_pango.so.6
        linux-vdso.so.1 =>  (0x00007ffec4fe2000)
        libgvc.so.6 => /root/miniconda3/lib/graphviz/../libgvc.so.6 (0x00007fba3fcc6000)
        libltdl.so.7 => /root/miniconda3/lib/graphviz/../libltdl.so.7 (0x00007fba3fabd000)
        libxdot.so.4 => /root/miniconda3/lib/graphviz/../libxdot.so.4 (0x00007fba3f8b8000)
        libcgraph.so.6 => /root/miniconda3/lib/graphviz/../libcgraph.so.6 (0x00007fba3f6a2000)
        libcdt.so.5 => /root/miniconda3/lib/graphviz/../libcdt.so.5 (0x00007fba3f49c000)
        libpathplan.so.4 => /root/miniconda3/lib/graphviz/../libpathplan.so.4 (0x00007fba3f294000)
        libexpat.so.1 => /root/miniconda3/lib/graphviz/../libexpat.so.1 (0x00007fba3f06a000)
        libpangocairo-1.0.so.0 => /root/miniconda3/lib/graphviz/../libpangocairo-1.0.so.0 (0x00007fba3ee5e000)
        libcairo.so.2 => /root/miniconda3/lib/graphviz/../libcairo.so.2 (0x00007fba3eb19000)
        libpixman-1.so.0 => /root/miniconda3/lib/graphviz/../libpixman-1.so.0 (0x00007fba3e873000)
        libXrender.so.1 => not found
        libSM.so.6 => not found
        libICE.so.6 => not found
        libX11.so.6 => not found
        libXext.so.6 => not found
        libpangoft2-1.0.so.0 => /root/miniconda3/lib/graphviz/../libpangoft2-1.0.so.0 (0x00007fba3e65e000)
        libharfbuzz.so.0 => /root/miniconda3/lib/graphviz/../libharfbuzz.so.0 (0x00007fba3e3d0000)
        libgraphite2.so.3 => /root/miniconda3/lib/graphviz/../libgraphite2.so.3 (0x00007fba3e19c000)
        libpango-1.0.so.0 => /root/miniconda3/lib/graphviz/../libpango-1.0.so.0 (0x00007fba3df52000)
        libgthread-2.0.so.0 => /root/miniconda3/lib/graphviz/../libgthread-2.0.so.0 (0x00007fba3dd51000)
        libfontconfig.so.1 => /root/miniconda3/lib/graphviz/../libfontconfig.so.1 (0x00007fba3db0f000)
        libxml2.so.2 => /root/miniconda3/lib/graphviz/../libxml2.so.2 (0x00007fba3d7a9000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007fba3d5a5000)
        libicui18n.so.58 => /root/miniconda3/lib/graphviz/../libicui18n.so.58 (0x00007fba3d0a5000)
        libicuuc.so.58 => /root/miniconda3/lib/graphviz/../libicuuc.so.58 (0x00007fba3cca6000)
        libicudata.so.58 => /root/miniconda3/lib/graphviz/../libicudata.so.58 (0x00007fba3b1a6000)
        liblzma.so.5 => /root/miniconda3/lib/graphviz/../liblzma.so.5 (0x00007fba3af80000)
        libgobject-2.0.so.0 => /root/miniconda3/lib/graphviz/../libgobject-2.0.so.0 (0x00007fba3ad2b000)
        libffi.so.6 => /root/miniconda3/lib/graphviz/../libffi.so.6 (0x00007fba3ab22000)
        libglib-2.0.so.0 => /root/miniconda3/lib/graphviz/../libglib-2.0.so.0 (0x00007fba3a7f7000)
        libiconv.so.2 => /root/miniconda3/lib/graphviz/../libiconv.so.2 (0x00007fba3a509000)
        libpcre.so.1 => /root/miniconda3/lib/graphviz/../libpcre.so.1 (0x00007fba3a2c7000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007fba3a0aa000)
        librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007fba39ea2000)
        libfreetype.so.6 => /root/miniconda3/lib/graphviz/../libfreetype.so.6 (0x00007fba39c0a000)
        libpng16.so.16 => /root/miniconda3/lib/graphviz/../libpng16.so.16 (0x00007fba399ca000)
        libz.so.1 => /root/miniconda3/lib/graphviz/../libz.so.1 (0x00007fba397b3000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fba394aa000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fba390e0000)
        /lib64/ld-linux-x86-64.so.2 (0x00007fba4016c000)
        libXrender.so.1 => not found
        libSM.so.6 => not found
        libICE.so.6 => not found
        libX11.so.6 => not found
        libXext.so.6 => not found
        libXrender.so.1 => not found
        libSM.so.6 => not found
        libICE.so.6 => not found
        libX11.so.6 => not found
        libXext.so.6 => not found
        libstdc++.so.6 => /root/miniconda3/lib/graphviz/.././libstdc++.so.6 (0x00007fba38da6000)
        libgcc_s.so.1 => /root/miniconda3/lib/graphviz/.././libgcc_s.so.1 (0x00007fba38b94000)
```

If I install them from conda-forge it works.

```
FROM ubuntu:latest

RUN apt-get update && apt-get -y dist-upgrade
RUN apt-get install -y --no-install-recommends bzip2 wget

RUN wget --no-check-certificate https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
    bash Miniconda3-latest-Linux-x86_64.sh -bf && \
    rm Miniconda3-latest-Linux-x86_64.sh
ENV PATH="/root/miniconda3/bin:${PATH}"

RUN conda install -c conda-forge graphviz=2.38.0 xorg-libxrender xorg-libxpm

CMD dot
```

I don't know if they should be added as dependencies or if a fancier solution is required?

**Comment by jakirkham** [comment]
Did you install the yum requirements from the recipe? Note the names may be different for `apt-get`.

**Comment by jakirkham** [comment]
Just install everything listed in `yum_requirements.txt`. There is currently no `gtk2` package. Using X11 `conda`packages as dependencies is an ongoing discussion.

**Comment by rasmusagren** [comment]
I understand. May I ask why you're reluctant to add it as a dependency? Is it because conda-forge doesn't want large dependencies that the majority of the users would already have installed? If so, is there a list somewhere of these programs that are considered "outside of conda"? Thanks!

**Comment by ccordoba12** [comment]
@rasmusagren, the simplest way for you to solve this for now is to install an X server in your server.

**Comment by rasmusagren** [comment]
Thanks, but the solution with xorg-libxrender xorg-libxpm works for me right now. I still think that all dependencies should be included in the recipes, in particular as docker and singularity are becoming so popular and there people often use slimmed down distributions.

**Comment by jakirkham** [comment]
Should add put up PR ( https://github.com/conda-forge/staged-recipes/pull/3920 ) for `gtk2` some time ago, which is the only package we are getting from `yum` here. Not sure if that is enough to make this installable from `conda` alone or if there are more issues hidden.

**Comment by jakirkham** [comment]
> So, no gtk needed.

Ha! [No wonder]( https://github.com/conda-forge/graphviz-feedstock/blob/046c1d35d8c8f6042d253d74220a323c1c5b388b/recipe/build.sh#L23 ). :)

**Comment by ccordoba12** [comment]
Fixed by #23.
